src/resources/postcss/tribe-common-admin/settings/_props.pcss

```diff
@@ -21,4 +21,7 @@
 	/** new styles to go in utilities */
 	--tec-border-radius-settings: 6px;
 	--tec-width-settings-form: 900px;
+
+	/** Font styles */
+	--tec-font-family-arial: Arial, sans-serif;
```

Comment:
```suggestion
```

nope - we have one, let's use it:
`--tec-font-family-sans-serif: "Helvetica Neue", Helvetica, -apple-system, BlinkMacSystemFont, Roboto, Arial, sans-serif;`

I don't want to add a new font family - this one is well tested and covers a lot of edge cases.

Reply:
This was to match the design, where the font was specifically supposed to be Arial.